### PR TITLE
Add route to list permissions for docs by IDs

### PR DESCRIPTION
**Pull request by aenario**

### CODE REVIEW

**Comment by aenario** [comment]
Let's merge as-is... More complex permissions (for albums) will come in another PR.

File: docs/permissions.md [hunk 1]
```diff
@@ -418,3 +418,46 @@ Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3O
 ```http
 HTTP/1.1 204 No Content
 ```
+
+### POST /permissions/exists
```
Comment (nono) [inline docs/permissions.md]: Why a `POST` and not a `GET`?

Comment (aenario) [inline docs/permissions.md]: The list of IDs we want to fetch the sharing status for can be quite long, so we have to pass it in the body. GET request cannot have URLS...

**Comment by aenario** [comment]
@jinroh ?

**Pull request merged**